repos/system_upgrade/common/actors/checksaphana/libraries/checksaphana.py

```diff
@@ -8,7 +8,8 @@
 
 SAP_HANA_MINIMAL_MAJOR_VERSION = 2
 SAP_HANA_RHEL8_REQUIRED_PATCH_LEVELS = ((5, 54, 0),)
```

Comment:
(5, 54, 0) is accurate if the upgrade is from 7.9 to 8.2
if it's from 7.9 to 8.6, then HANA must be withing a version of  [59.02; 59.05]